.gitignore

```diff
@@ -136,7 +136,7 @@ dmypy.json
 .DS_Store
 
 # Crawler artifacts
-tmp
+tmp/pages
```

Comment:
because of this change, I see 422 files changed (!). I cannot review that before you remove these files from the index.